install/netbox-install.sh

```diff
@@ -0,0 +1,125 @@
+msg_ok "Set up PostgreSQL"
+
+msg_info "Installing NetBox"
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/netbox-community/netbox/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+wget -q "https://github.com/netbox-community/netbox/archive/refs/tags/v${RELEASE}.zip"
+unzip -q "v${RELEASE}.zip"
+ln -s "/opt/netbox-${RELEASE}/" /opt/netbox
+
+$STD adduser --system --group netbox
+chown --recursive netbox /opt/netbox/netbox/media/
+chown --recursive netbox /opt/netbox/netbox/reports/
+chown --recursive netbox /opt/netbox/netbox/scripts/
+
+mv /opt/netbox/netbox/netbox/configuration_example.py /opt/netbox/netbox/netbox/configuration.py
+
+SECRET_KEY=$(python3 /opt/netbox/netbox/generate_secret_key.py)
+ESCAPED_SECRET_KEY=$(printf '%s\n' "$SECRET_KEY" | sed 's/[&/\]/\\&/g')
+
+sed -i 's/ALLOWED_HOSTS = \[\]/ALLOWED_HOSTS = ["*"]/' /opt/netbox/netbox/netbox/configuration.py
+sed -i "s|SECRET_KEY = ''|SECRET_KEY = '${ESCAPED_SECRET_KEY}'|" /opt/netbox/netbox/netbox/configuration.py
+sed -i "/DATABASE = {/,/}/s/'USER': '[^']*'/'USER': '$DB_USER'/" /opt/netbox/netbox/netbox/configuration.py
+sed -i "/DATABASE = {/,/}/s/'PASSWORD': '[^']*'/'PASSWORD': '$DB_PASS'/" /opt/netbox/netbox/netbox/configuration.py
+
+$STD /opt/netbox/upgrade.sh &>/dev/null
+ln -s /opt/netbox/contrib/netbox-housekeeping.sh /etc/cron.daily/netbox-housekeeping
+
+mv /opt/netbox/contrib/apache.conf /etc/apache2/sites-available/netbox.conf
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/private/netbox.key -out /etc/ssl/certs/netbox.crt -subj "/C=US/O=NetBox/OU=Certificate/CN=localhost" &>/dev/null
+$STD a2enmod ssl proxy proxy_http headers rewrite
+$STD a2ensite netbox
+systemctl restart apache2
+
+cp /opt/netbox/contrib/gunicorn.py /opt/netbox/gunicorn.py
+cp /opt/netbox/contrib/*.service /etc/systemd/system/
+systemctl daemon-reload
+systemctl enable -q --now netbox netbox-rq
+
+msg_ok "Installed NetBox"
+
+msg_info "Setting up Django Admin"
+DJANGO_USER=Admin
+DJANGO_PASS=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | cut -c1-13)
+
+source /opt/netbox/venv/bin/activate
+$STD python3 /opt/netbox/netbox/manage.py shell << EOF
+from django.contrib.auth import get_user_model
+UserModel = get_user_model()
+user = UserModel.objects.create_user('$DJANGO_USER', password='$DJANGO_PASS')
+user.is_superuser = True
+user.is_staff = True
+user.save()
+EOF
+msg_ok "Setup Django Admin"
+
+msg_info "Save credential file"
+# PSQL:
+{
+echo "Netbox-Credentials"
+echo -e "Netbox Database User: \e[32m$DB_USER\e[0m"
+echo -e "Netbox Database Password: \e[32m$DB_PASS\e[0m"
+echo -e "Netbox Database Name: \e[32m$DB_NAME\e[0m"
+echo -e "Netbox Secret: \e[32m$SECRET_KEY\e[0m"
+} >> ~/netbox.creds
+
+# DJANGO:
+{
+echo ""
+echo "Netbox-Django-Credentials"
+echo -e "Django User: \e[32m$DJANGO_USER\e[0m"
+echo -e "Django Password: \e[32m$DJANGO_PASS\e[0m"
+} >> ~/netbox.creds
```

Comment:
Move this to `Setting up Django Admin` step

Reply:
Done